pyproject.toml

```diff
@@ -48,6 +48,7 @@ test = [
 packages = [
     "pymc_marketing",
     "pymc_marketing.mmm",
+    "pymc_marketing.mmm.models",
```

Comment:
adding pymc_marketing.mmm.models.components might be required and get you through your issue if I understood the issue correctly. Granted, I don't use setuptools much. 

One suggestion here: Stick with just a models submodule to simplify the imports. Is it required to have such a deep import?

- Before: `from pymc_marketing.mmm.models.components import ...`
- Alternative 1: `from pymc_marketing.mmm.models.lagging import ...`
- Alternative 2: `from pymc_marketing.mmm.lagging import ...`

Adding submodule comes with an extra `__init__.py` file and more folder structure which I find cumbersome